sdcm/logcollector.py

```diff
@@ -766,6 +766,18 @@ def archive_dir_to_tarfile(logdir):
             return None
         return archive_name
 
+    @staticmethod
+    def archive_file_to_tarfile(file_path: str) -> str:
+        filename = os.path.basename(file_path)
+        archive_name = f"{filename}.tar.gz"
+        try:
+            with tarfile.open(archive_name, "w:gz") as tar:
+                tar.add(file_path, arcname=filename)
+        except Exception as details:
+            LOGGER.error("Error during archive creation. Details: \n%s", details)
+            return None
+        return archive_name
+
```

Comment:
It is complete duplicate of above "archive_dir_to_tarfile" method.
If you provide there path to a file, it will work the same way.

So, I guess proper approach would be here to rename existing method from "archive_dir_to_tarfile" to "archive_to_tarfile" and upadte it's argument name to something like "dir_or_file_path".

Reply:
You absolutely right. Forgot to do that. fixed